packages/grafana-ui/src/components/Badge/Badge.story.tsx

```diff
@@ -3,7 +3,7 @@ import { boolean, text, select } from '@storybook/addon-knobs';
 import { Badge, BadgeColor } from './Badge';
 
 export default {
-  title: 'Other/Badge',
+  title: 'Layout/Badge',
```

Comment:
Don't think this is Layout

Reply:
I agree. Not sure which category it would be though. If you ask me, anything is better than Other

Reply:
We could skip this change for now though, I want to get the fix for StatsPicker merged.

Reply:
`Data disaplay` as of ant.d?

Reply:
We'll go with that for now, but it definitely needs to be discussed.